src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/HistoricalBeneficialOwnerValidatorTest.java

```diff
@@ -182,6 +182,15 @@ void testErrorReportedWhenCeasedDateFieldIsNull() {
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
+    @Test
+    void testNoErrorReportedWhenCeasedDateFieldIsNullAndIsUnableToObtainAllTrustInfoIsSetToTrue() {
```

Comment:
is there a test for the opposite situation, where an error is reported if ceased date is null and unable_to_obtain_info is false

Reply:
Yes, it's covered in another test that's not explicitly setting the value to false as that is the default value. Not sure what the value is in adding another test as this scenario is already covered.

Reply:
i think i'm just being too cautious, i'm ok with the default.